android/app/build.gradle

```diff
@@ -81,9 +86,15 @@ dependencies {
     implementation 'com.google.firebase:firebase-crashlytics-ndk'
   }
 
-  implementation 'androidx.car.app:app:1.4.0-alpha01'
-  // Fix for app/organicmaps/util/FileUploadWorker.java:14: error: cannot access ListenableFuture
-  implementation 'com.google.guava:guava:29.0-android'
+  if (androidAutoEnabled) {
+    println('Building with Android Auto')
+    // Fix for app/organicmaps/util/FileUploadWorker.java:14: error: cannot access ListenableFuture
+    implementation 'com.google.guava:guava:29.0-android'
+    implementation 'androidx.car.app:app:1.4.0-alpha01'
```

Comment:
beta01

Reply:
beta01 requires upgrading `propCompileSdkVersion` from 33 to 34 and Android Studio recommends to update the gradle plugin for it

```
This Android Gradle plugin (8.1.0) was tested up to compileSdk = 33 (and compileSdkPreview = "UpsideDownCakePrivacySandbox").

You are strongly encouraged to update your project to use a newer
Android Gradle plugin that has been tested with compileSdk = 34.
```

It's better to make these changes in master first

Reply:
Let's do it then )

Reply:
Please read the doc:

> 

    Note: 1.4-beta01 requires compileSdk34, which is still in dev status. 1.4-beta02 will correct this issue. Alternatively, you can add android.suppressUnsupportedCompileSdk=34 to your project's settings.gradle file to safely temporarily suppress the warning.

https://developer.android.com/jetpack/androidx/releases/car-app

34 is not ready yet.

Reply:
upgraded to beta02.